docs/content/node/getting-started/quickstart/data/data.md

```diff
@@ -19,4 +19,47 @@ endswithURL: >
 expensesPath1: examples/quickstart/polar/expenses-01-nodejs.polar
 expensesPath2: examples/quickstart/polar/expenses-02-nodejs.polar
 isAllowed: isAllowed
+installation_new: |
+    ```bash
+    npm install --save oso
+    ```
+    Or find the Oso package at [http://pypi.python.org/pypi/oso/](http://pypi.python.org/pypi/oso/).
+import: import
+import_code: |
+    ```js
+    const { Oso } = require('oso')
+    oso = Oso()
+    oso.enableRoles()
```

Comment:
TODO: make sure we sync this up w/ @saolsen's ongoing validation work. In the latest iteration of that work, you need to call `enableRoles()` _after_ your config policy is loaded b/c we perform validation in `enableRoles()`.